### PR TITLE
set default tag for all resources pointing to repository

### PR DESCRIPTION
**Pull request by afeld**
New Terraform provider feature:

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging#propagating-tags-to-all-resources

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -26,6 +26,12 @@ terraform {
 provider "aws" {
   # arbitrary, since most of these resources are global
   region = "us-east-1"
+
+  default_tags {
+    tags = {
+      Project = "https://github.com/18F/aws-admin"
```
Comment (afeld) [inline terraform/main.tf:32]: This is the important part.

**Comment by afeld** [comment]
Build error is unrelated: https://github.com/18F/aws-admin/issues/136

**Review by JJediny — APPROVED** [review]
lg

**Comment by afeld** [comment]
Build failure due to https://github.com/18F/aws-admin/issues/136.

**Comment by afeld** [comment]
@JJediny I won't `apply` this since I know you're working on https://github.com/18F/aws-admin/issues/138.

**Pull request merged**